rbac/management/access/view.py

```diff
@@ -119,3 +137,13 @@ def get_paginated_response(self, data):
         """Return a paginated style `Response` object for the given output data."""
         assert self.paginator is not None
         return self.paginator.get_paginated_response(data)
+
+    def validate_and_get_param(self, params):
+        """Validate input parameters and get ordering and sub_key."""
+        validate_limit_and_offset(params)
+        app = params.get(APPLICATION_KEY)
+        sub_key = app
+        ordering = validate_and_get_key(params, ORDER_FIELD, VALID_ORDER_VALUES, "unordered")
```

Comment:
Can/should we make `order_by` optional? Just thinking for consistency, like where `http://localhost:8000/api/rbac/v1/roles/?order_by=` doesn't fail, it just doesn't order, the same as if you supplied `?foo=` would have no effect.

Reply:
Yea, after modifying the util function, it is as desired